specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlPool.json

```diff
@@ -4459,6 +4975,149 @@
+          "type": "integer"
+        },
+        "maxResourcePercent": {
+          "format": "int32",
+          "description": "The workload group cap percentage resource.",
+          "type": "integer"
+        },
+        "minResourcePercentPerRequest": {
+          "format": "double",
+          "description": "The workload group request minimum grant percentage.",
+          "type": "number"
+        },
+        "maxResourcePercentPerRequest": {
+          "format": "double",
+          "description": "The workload group request maximum grant percentage.",
+          "type": "number"
+        },
+        "importance": {
+          "description": "The workload group importance level.",
+          "type": "string"
+        },
+        "queryExecutionTimeout": {
+          "format": "int32",
+          "description": "The workload group query execution timeout.",
+          "type": "integer"
+        }
+      }
+    },
+    "WorkloadGroup": {
+      "description": "Workload group operations for a SQL pool.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/WorkloadGroupProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "WorkloadGroupListResult": {
+      "description": "A list of workload groups.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WorkloadGroup"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "WorkloadClassifierProperties": {
+      "description": "Workload classifier definition. For more information look at sys.workload_management_workload_classifiers (DMV).",
+      "required": [
+        "memberName"
+      ],
+      "type": "object",
+      "properties": {
+        "memberName": {
+          "description": "The workload classifier member name.",
+          "type": "string"
+        },
+        "label": {
+          "description": "The workload classifier label.",
+          "type": "string"
+        },
+        "context": {
+          "description": "The workload classifier context.",
+          "type": "string"
+        },
+        "startTime": {
+          "description": "The workload classifier start time for classification.",
+          "type": "string"
+        },
+        "endTime": {
+          "description": "The workload classifier end time for classification.",
+          "type": "string"
+        },
+        "importance": {
```

Comment:
same string enum comment as the other importance property

Reply:
There are five levels of importance: low, below_normal, normal, above_normal, and high.